blog-service/2024-12-24-apps.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Apps, Solutions, and Collection Integrations - December Release (Observability)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - apps
+  - releases-notes
+hide_table_of_contents: true  
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>
+
+### New release
+
+We’re excited to announce the release of new Azure Service Bus, Azure API Management, and AWS Serverless Application Models for Sumo Logic.
+
+- **Azure Service Bus**. Azure Service Bus is a fully managed enterprise message broker with message queues and publish-subscribe topics (in a namespace) used for decoupling applications and services from each other. This integration helps in monitoring incoming/outgoing messages, connections, throttled requests, and resource usage of your Service Bus namespace. [Learn more](/docs/integrations/microsoft-azure/azure-service-bus/).
+
+- **Azure API Management**. Azure API Management is a hybrid multicloud platform for managing APIs across different environments. As a platform-as-a-service, it supports the entire API lifecycle and provides near real-time visibility into API status and health, offering essential API Management operations and details for auditing. [Learn more](/docs/integrations/microsoft-azure/azure-api-management/).
+
+- **AWS Serverless Application Models**. Released the following four SAM’s with Python v3.13 AWS Lambda runtime update with latest libraries.
+
+    - sumologic-securityhub-connector-aws-org - SAM SemanticVersion: 1.0.8.
```

Comment:
```suggestion
    - `sumologic-securityhub-connector-aws-org`. Upgraded the SAM SemanticVersion to `v1.0.8`.
```

Reply:
Remove the linespace between the bullets and follow the same changes to other points.